templates/rancher/manifest.yaml

```diff
@@ -9,8 +9,8 @@ variables:
     optional: true
     description: "Specify rancher image tag for the latest commit/version of rancher."
   bootstrap_password:
-    readOnly: true
     type: string
+    default: password
```

Comment:
I this is fine to have be able to set, but I disagree on having it default to password. The default should be whatever rancher sets.

Reply:
I can remove the `default` from the variable, but I still need it to not be `readOnly`. That is to match the Github actions environment setup.

Reply:
I agree, I didn't say remove it. I'm just saying the default should be what rancher sets.